pkg/crc/validation/validation.go

```diff
@@ -79,14 +79,15 @@ func ValidateBundle(bundlePath string, preset crcpreset.Preset) error {
 func bundleMismatchWarning(userProvidedBundle string, preset crcpreset.Preset) {
 	userProvidedBundle = bundle.GetBundleNameWithExtension(userProvidedBundle)
 	if userProvidedBundle != constants.GetDefaultBundle(preset) {
+		warnMsg := fmt.Sprintf("Using %s bundle, but %s is expected for this release", userProvidedBundle, constants.GetDefaultBundle(preset))
 		// Should append underscore (_) here, as we don't want crc_libvirt_4.7.15.crcbundle
 		// to be detected as a custom bundle for crc_libvirt_4.7.1.crcbundle
 		usingCustomBundle := strings.HasPrefix(bundle.GetBundleNameWithoutExtension(userProvidedBundle),
 			fmt.Sprintf("%s_", bundle.GetBundleNameWithoutExtension(constants.GetDefaultBundle(preset))))
 		if usingCustomBundle {
-			logging.Warnf("Using custom bundle %s", userProvidedBundle)
+			warnMsg = fmt.Sprintf("Using custom bundle %s", userProvidedBundle)
 		}
-		logging.Warnf("Using %s bundle, but %s is expected for this release", userProvidedBundle, constants.GetDefaultBundle(preset))
+		logging.Warn(warnMsg)
```

Comment:
Why not if/else? This feels easier to read rather than having to look up what `warnMsg` is set to after reaching the end of the code?

```
if usingCustomBundle {
	logging.Warnf("Using custom bundle %s", userProvidedBundle)
} else {
	logging.Warnf("Using %s bundle, but %s is expected for this release", userProvidedBundle, constants.GetDefaultBundle(preset))
}
```